include/bluetooth/mesh/light_ctrl.h

```diff
@@ -183,6 +184,44 @@ enum bt_mesh_light_ctrl_coeff {
 	BT_MESH_LIGHT_CTRL_COEFF_KPU = BT_MESH_PROP_ID_LIGHT_CTRL_REG_KPU,
 };
 
+/** @brief Get the sensor format for the given Light Lightness Control property ID.
+ *
+ *  @param[in] id Light Lightness Control property or coefficient ID.
+ *
+ *  @return The associated sensor format, or NULL if the ID is unknown.
+ */
+static inline const struct bt_mesh_sensor_format *
+bt_mesh_lc_prop_format_get(int id)
```

Comment:
`uint16_t`? Makes it more obvious that this is the Property ID from the spec/Device Properties document.